.evergreen/generated_configs/functions.yml

```
@@ -180,10 +180,7 @@ functions:
         working_dir: drivers-evergreen-tools
         args:
           - -c
-          - |
-            for file in $(find .evergreen -type f -name \*.sh); do
-                chmod +x "$file" || exit
-            done
+          - find .evergreen -type f -name "*.sh" -execdir chmod +x "{}" \;
```

```suggestion
          - find .evergreen -type f -name "*.sh" -execdir chmod +rx "{}" \;
```

Add the `+r` here too, for consistency.

---

Done. `functions.yml` is generated. I forgot to regenerate.